README.md

```diff
@@ -37,14 +37,14 @@ maxSurge: 0
 
 The Deployment controller will bring the old replica down before starting the new one. The Nx Cloud client (that is running your tasks on your CI) will wait for the new container to boot, so no downtime will occur.
 
-Second, NX_CLOUD_APP_URL is the domain of the external IP of the LoadBalancer. You use it when running: `NX_CLOUD_API=http://some-ip-or-domain nx connect-to-nx-cloud`.
+Second, `NX_CLOUD_APP_URL` is the domain of the external IP of the LoadBalancer. You use it when running: `NX_CLOUD_API=http://some-ip-or-domain npx nx connect-to-nx-cloud`.
 
 Third, if you use AWS or Azure to store cached artifacts, and you don't run with the MongoDB embedded, Nx Cloud doesn't need a persistent volume.
 
-Fourth, if you are using an embedded file server, the PersistentVolumeClaim's accessMode affects how many replicas you can run. If it's set to ReadWriteMany, you can run many replicas. If it's set to ReadWriteOne, you can run many replicas.
+Fourth, if you are using an embedded file server, the PersistentVolumeClaim's accessMode affects how many replicas you can run. If it's set to ReadWriteMany, you can run many replicas. If it's set to ReadWriteOne, you can run one replica.
```

Comment:
I'm not sure if I understand this correctly. Is it "one replica" that is supposed to be here?